test/modules/fundingManager/bondingCurve/utils/mocks/RedeemingBondingCurveBaseV1Mock.sol

```diff
@@ -97,6 +113,14 @@ contract RedeemingBondingCurveBaseV1Mock is RedeemingBondingCurveBase_v1 {
     //--------------------------------------------------------------------------
     // Mock access for internal functions
 
+    function call_calculateSaleReturn(uint _depositAmount)
```

Comment:
What does this do? `calculateSaleReturn is public anyways, why the wrapper?

Reply:
This is for the call of the internal function.
Yes we technically just expose it via public, but we still test it as internal.
This is probably up to discussion.
I honestly dont know why this popped up in the PR, as the tests written for this werent touched by me :man_shrugging:

Reply:
Cleared up after discussion:
Was a leftover from a refactoring that happend a while ago.
I moved it because it was in the wrong section and thats why it came up.
I now removed the function and let the tests call it directly to clean it up

Reply:
Done